valhalla/baldr/admin.h

```diff
@@ -1,6 +1,7 @@
 #ifndef VALHALLA_BALDR_ADMIN_H_
 #define VALHALLA_BALDR_ADMIN_H_
 
+#include <algorithm>
```

Comment:
if we only need access to this in admin.cc then put the include there. better to keep our headers as slim as possible so that people using our headers dont get more stuff than they need (not that it bloats their binaries but just because we should advertize only what we need to, ie only what is used directly in this header file)